internal/api/mfa.go

```diff
@@ -40,6 +43,11 @@ type EnrollFactorResponse struct {
 	Type         string     `json:"type"`
 	FriendlyName string     `json:"friendly_name"`
 	TOTP         TOTPObject `json:"totp,omitempty"`
+	PhoneNumber  string     `json:"phone_number,omitempty"`
```

Comment:
can we make this `phone` instead for consistency with phone signups / sign-ins?
```suggestion
	PhoneNumber  string     `json:"phone,omitempty"`
```